doc/api/process.md

```diff
@@ -2009,6 +2009,49 @@ if (process.getgroups) {
 This function is only available on POSIX platforms (i.e. not Windows or
 Android).
 
+## `process.getOptionValue(option)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
```

Comment:
Given the experimental nature of many of the individual options, wouldn't this be Experimental forever? Ergo, cannot be relied upon.